mlir/lib/Dialect/Vector/IR/VectorOps.cpp

```diff
@@ -170,12 +170,37 @@ AffineMap mlir::vector::getTransferMinorIdentityMap(ShapedType shapedType,
       shapedType.getContext());
 }
 
+static bool couldBeSameValueWithMasking(vector::TransferWriteOp defWrite,
+                                        vector::TransferReadOp read) {
+  if (!defWrite.getMask() && !read.getMask())
+    return true; // Success: No masks (values will be the same).
```

Comment:
```suggestion
  if (!defWrite.getMask() && !read.getMask()) {
     // Success: No masks (values will be the same).
    return true;
 }
```
I think that this would be more "canonical" in LLVM. This is a nit, feel free to ignore.